code/Test_definitions/carrier-billing-confirmPayment.feature

```diff
@@ -0,0 +1,197 @@
+Feature: CAMARA Carrier Billing API, v0.3 - Operation confirmPayment
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * Endpoint required for 2-step payment
+  #
+  # Testing assets:
+  # * A phone number eligible for payment (no restrictions for it to be used to perform a payment)
+  # * A phone number not-eligible for payment (payment is denied for it due to business conditions)
+  #
+  # References to OAS spec schemas refer to schemas specifies in carrier-billing.yaml, version 0.3.0
+
+  Background: Common confirmPayment setup
+    Given the resource "/carrier-billing/v0.3/payments/{paymentId}/confirm"                                                              |
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the path parameter "paymentId" is set to a valid value
+    And the request body is set by default to a request body compliant with the schema
+
+  ##############################
+  # Happy path scenarios
+  ##############################
+
+
+  @confirm_payment_01_generic_success_scenario
+  Scenario: Common validations for any success scenario
+    # Valid default request body compliant with the schema
```

Comment:
Same than for cancel operation. I was expecting a `given` clause.

Reply:
idem as above